BUGFIX: Add missing backticks (#184)
TYPO3-Documentation/TYPO3CMS-Tutorial-GettingStarted: Documentation/Installation/Install.rst

           ddev composer create "typo3/cms-base-distribution:^11"

> This command pulls down the latest release of TYPO3 and places it in the directory specified ``YourProjectDirectory``.

Reviewer: ```suggestion
This command pulls down the latest release of TYPO3 and places it in the :file:`example-project-directory`.
```

> After this command has finished running, ``example-project-directory`` will contain the following structure:

Reviewer: ```suggestion
After this command has finished running, :file:`example-project-directory` will contain the following structure:
```

In the event that certain criteria aren't met, TYPO3 will display a list of issues it has detected accompanied by a resolution for each issue.

> Once changes have been made, TYPO3 can re-scan the host environment by reloading the page ``https://example-project-site.local/typo3/install.php``.

Reviewer: ```suggestion
Once changes have been made, TYPO3 can re-scan the host environment by reloading the page `https://example-project-site.local/typo3/install.php`.
```

The list of databases available is dependent on which database drivers are installed on the host.

> For example, if an instance of TYPO3 is intended to be used with a MySQL database then the PHP extension ``pdo_mysql`` is required.

Reviewer: ```suggestion
For example, if an instance of TYPO3 is intended to be used with a MySQL database then the PHP extension `pdo_mysql` is required.
```

> Once it's installed, ``MySQL Database`` will be an available option.(Review)

Reviewer: ```suggestion
Once it's installed, :guilabel:`MySQL Database` will be an available option.(Review)
```
